app/widgets/common/deployModal/EnvironmentDropdown/EnvironmentDropdown.types.ts

```diff
@@ -0,0 +1,19 @@
+import type { Deployment } from '../../deploymentsView/types';
+
+export interface FetchedEnvironment {
+    id: string;
+    // eslint-disable-next-line
+    display_name: string;
+    capabilities: Deployment['capabilities'];
+}
+
+export interface Environment {
+    id: string;
+    capabilities: Deployment['capabilities'];
+    displayName: string;
+}
+
+export type FilteredEnvironments = {
+    suggestedEnvironments: Environment[];
+    nonSuggestedEnvironments: Environment[];
```

Comment:
I suggest (nomen omen 🙂) to have either:
```suggestion
    notSuggestedEnvironments: Environment[];
```
or
```suggestion
    otherEnvironments: Environment[];
```

Reply:
Issue has been addressed [here](https://github.com/cloudify-cosmo/cloudify-stage/pull/2396#discussion_r1083071238)